package.json

```diff
@@ -12,7 +12,7 @@
   },
   "dependencies": {
     "@ethersphere/bee-js": "^8.1.0",
-    "@solarpunkltd/comment-system-ui": "^0.0.12",
+    "@solarpunkltd/comment-system-ui": "file:/Users/seresroland/Desktop/DEVCONAPP/SwarmCommentSystem/comment-system-ui",
```

Comment:
set to version 0.0.13
make sure that the comment-ui npm package is first published